docs/tutorials/mlflow.ipynb

```diff
@@ -0,0 +1,819 @@
+{
```

Comment:
<p>There's a link in the results that points at <a href="https://dashboard.default-tenant.app.llm2.iguazio-cd0.com/mlprojects/mlflow-tracking-example/jobs/monitor/4185accc906648a9a56f73d97829c1d0/overview" target="_blank">https://dashboard.default-tenant.app.llm2.iguazio-cd0.com/mlprojects/mlflow-tracking-example/jobs/monitor/4185accc906648a9a56f73d97829c1d0/overview</a> - the actual llm2 system used to generate this. Better to remove that link, or replace it with something irrelevant.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4694/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
removed